src/main/java/domain/LottoRate.java

```diff
@@ -0,0 +1,23 @@
+package domain;
+
+import java.util.List;
+
+public class LottoRate {
+
+    private static final int INITIAL_NUMBER_ZERO = 0;
+    private static final int THREE_MATCHES = 5000;
+    private static final int FOUR_MATCHES = 50000;
+    private static final int FIVE_MATCHES = 150000;
+    private static final int SIX_MATCHES = 20000000;
+    private int sum = 0;
```

Comment:
```suggestion
    private static final int SIX_MATCHES = 20000000;
    
    private int sum = 0;
```

sum = 0을 생성자에서 해주는 건 어떨까요??